demisto_sdk/scripts/update_demisto_sdk_version.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Get CD path list from argument
+repositories=("$@")
+
+# Validate CD path list
+if [ ${#repositories[@]} -eq 0 ]; then
+  echo "Usage: $0 <repository1> <repository2> ..."
+  exit 1
+fi
+
+# Save current path
+current_path="$PWD"
+
+# Loop through paths
+for repository in "${repositories[@]}"
+do
+  # Validate path
+  if [ ! -d "$repository" ]; then
+    echo "Error: $repository is not a valid directory."
+    exit 1
+  fi
+
+  # CD to path
+  cd "$repository" || exit
```

Comment:
That way the repos should be given as absolute paths.
You could use `pushd` (instead of `cd`), and then the repos could be either absolute or relative